package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -170,7 +170,26 @@ append_address() {
 }
 
 append_ipset() {
```

Comment:
This change will only cover the "old way" of defining ipsets.

We have two option to define ipsets within the dnsmasq section:

```
config dnsmasq
	list   ipset '/openwrt.org/openwrt_ipv4,openwrt_ipv6'
```

And since acb336235cc640980ea549655b0702daf9e628f8 in addition

```
config ipset
            list name 'openwrt_ipv4'
            list name 'openwrt_ipv6'
            list domain 'openwrt.org'
```

May I propose something else:

1. migrate the old style to the new one
2. drop support for the old style
3. add the changes to `dnsmasq_ipset_add` instead

Based on Kevins changes I've already written the code doing so. Feel free to pick the changes from [my staging tree](https://git.openwrt.org/?p=openwrt/staging/mkresin.git).